packages/core/src/tracing/spanBuffer.js

```diff
@@ -167,6 +183,13 @@ exports.addSpan = function (span) {
     return;
   }
 
+  // CASE: if isFaaS we no longer want to buffer spans after we have already send the bundle
+  if (isFaaS && transmitImmediate) {
```

Comment:
_question:_ Should we drop the `isFaas` part of the condition? If any code that is using `spanBuffer` sets `transmitImmediate` to true, the expectation is probably that spans are transmitted immediately, no matter which environment we are in. At the moment we only need this for FAAS, so it doesn't really matter. Still, IMO there is no reason to check `isFaas` here, checking `transmitImmediately` is enough I think. WDYT?

Reply:
Yes was thinking about that too 👍